.vscode/launch.json

```diff
@@ -8,7 +8,7 @@
       "name": "Python: Remote Attach (Outside container)",
       "type": "python",
       "request": "attach",
-      "connect": { "host": "localhost", "port": 5688 },
+      "connect": { "host": "localhost", "port": 5686 },
```

Comment:
is this meant to be included?

Reply:
There is no debugging enabled for assessment on docker-runner yet. Once this merged we can add the config there like how it is done for other services(like frontend. fund-store...)